bzt/modules/blazemeter.py

```diff
@@ -430,6 +446,34 @@ def get_monitoring_json(self, session_id, user_id, test_id):
             "results": results
         }
 
+    def get_metrics_json(self):
+        datapoints = {}
+
+        for source, buff in iteritems(self.data):
+            for timestamp, item in iteritems(buff):
+                if source == 'local':
+                    source = platform.node()
+
+                if timestamp not in datapoints:
+                    datapoints[timestamp] = {}
+
+                for field, value in iteritems(item):
+                    if field in ('ts', 'interval'):
+                        continue
+                    if field.lower().startswith('cpu'):
```

Comment:
This is an artificial limiter to send only one metric (CPU), mostly for easy debugging.